config/.coderabbit.yaml

```
@@ -0,0 +1,251 @@
+    enabled: true
+    style: "google"
+    path_instructions:
+      - path: "src/**/*.py"
+        instructions: |
+          Generate Google-style docstrings with:
+          - Args section for parameters with types
+          - Returns section with type
+          - Raises section for exceptions
+          - Examples section for complex functions
+          Do not add "Auto-generated by CodeRabbit" notices.
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)